tests/func/test_analytics.py

```diff
@@ -1,65 +0,0 @@
-import os
```

Comment:
So we've removed these tests and didn't introduce new ones instead? Such a big changes need to be carefully covered by lots of tests, as it will easily break the built up trash that we've been testing in real world for over 1 year.

Reply:
Yes, should've marked it as a WIP